feature-group-definitions/aborting.dist.yml

```diff
@@ -25,3 +25,4 @@ compat_features:
   - api.AbortSignal
   - api.AbortSignal.abort_event
   - api.AbortSignal.aborted
+descriptionHTML: The <strong>AbortController</strong> and <code>AbortSignal</code> APIs allow you to cancel an ongoing operation, such as a <code>fetch()</code> request.
```

Comment:
I couldn't figure out how to insert `descriptionHTML` right after `description`. But it won't matter if we trim dist files into just the computed `status` and `compat_features`.

Reply:
It's a little fussy. You have to create a document node, then splice the document's items array to insert it near a target item. See `insertStatus` for an example: https://github.com/web-platform-dx/web-features/blob/a7e9e1c058519fda5986864cade005fc8a739aea/scripts/dist.ts#L173